assets/js/googlesitekit/datastore/user/user-info.js

```diff
@@ -398,6 +398,21 @@ export const selectors = {
 		return user !== undefined ? user.picture : user;
 	} ),
 
+	/**
+	 * Gets the full name name for this user.
+	 *
+	 * Returns full name of the user or `undefined` if the user info is not available/loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(string|undefined)} The user's full name.
+	 */
+	getFullName: createRegistrySelector( ( select ) => () => {
+		const user = select( CORE_USER ).getUser();
+		return user !== undefined ? user.full_name : user;
```

Comment:
This is a bit tough to read, because the comment reads like this will return `undefined` if `user` is not defined, but then returns `user`… which of course _is_ `undefined` but I think it requires a lot more thinking to parse what this code is actually doing.

Elsewhere we explicitly return `undefined` and return early instead of using a ternary, eg: https://github.com/google/site-kit-wp/blob/a38c9ba24eeaf8f1297e99a689dfeb6b5bb545b5/assets/js/components/RecoverableModules.js#L42-L44

Let's do that here too:

```suggestion
		const user = select( CORE_USER ).getUser();
		
		if ( user === undefined ) {
			return undefined;
		}
		
		return user.full_name;
```

Reply:
Yup - makes sense 👍